python/createCollectionDownload.py

```diff
@@ -80,7 +194,7 @@ def download_file(url:str, output_file:str):
                 #if chunk: 
                 f.write(chunk)
 
-    print(f"Saved {url}\nto\n{output_file}")
```

Comment:
I think we should keep the url in the print statement here, otherwise you can't copy+paste to try to download the file (without re-running the whole script and printing/logging the url of the generated download.

Reply:
I moved the URL print to the end of `download_collection()` because it would print twice and fill the console, but I can put it back here